src/debugger/Debugger.js

```diff
@@ -75,15 +72,12 @@ export class Debugger {
           lastPoll = Date.now();
         }
       }
-      fs.writeFileSync("./src/debugger/.sessionlogs/proxy2debugger.txt", "");
+      this.channel.writeOut("");
     }
   }
 
-  sendDebugInfo(ast: BabelNode, contextStack: Array<ExecutionContext>, lineNum: number) {
-    fs.writeFileSync("./src/debugger/.sessionlogs/debugger2proxy.txt", "breakpoint " + lineNum);
-    // for (let i = 0; i < contextStack.length; i++) {
-    //   let ctxt = contextStack[i];
-    // }
+  sendDebugInfo(ast: BabelNode, lineNum: number) {
+    this.channel.writeOut("breakpoint " + lineNum);
   }
 
   parseCommands(commands: Array<string>) {
```

Comment:
It is still not clear to me why there would be more than one command in a particular interaction.